src/radical/pilot/agent/scheduler/continuous.py

```diff
@@ -98,6 +98,12 @@ def _configure(self):
         # NOTE: for non-oversubscribing mode, we reserve a number of cores
         #       for the GPU processes - even if those GPUs are not used by
         #       a specific workload.
+
+        # MING: What do we do when nodes are oversubscribed? Do we use
+        #       something else?
```

Comment:
Yeah, we simply don't reserve cores.  Given your comment that seems not obvious from the `NOTE` above.  Any suggestion how to improve it?